src/main/java/org/folio/search/service/reindex/jdbc/ClassificationRepository.java

```diff
@@ -197,8 +197,12 @@ protected RowMapper<Map<String, Object>> rowToMapMapper() {
   }
 
   @Override
-  public void deleteByInstanceIds(List<String> instanceIds) {
-    var sql = DELETE_QUERY.formatted(JdbcUtils.getSchemaName(context), getParamPlaceholderForUuid(instanceIds.size()));
+  public void deleteByInstanceIds(List<String> instanceIds, String tenantId) {
+    var sql = DELETE_QUERY.formatted(
+      JdbcUtils.getSchemaName(context),
+      getParamPlaceholderForUuid(instanceIds.size()),
+      tenantId == null ? "" : "AND tenant_id = '%s'".formatted(tenantId));
```

Comment:
We have to use prepared queries. Add tenantId to the list of query params
```suggestion
      tenantId == null ? "" : "AND tenant_id = '?";
```